### PR TITLE
Some fixes and suggested changes

### PR DESCRIPTION
**Pull request by stetard**

### CODE REVIEW

File: Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx [hunk 1]
```diff
@@ -13,24 +13,24 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Anonymize.Confirm" xml:space="preserve">
-    <value>Vos données personnelles seront anonymisées. Après l'anonymisation, les données ne peuvent plus vous être identifiées personnellement et vous ne pourrez plus vous connecter à l'aide de ce compte. Souhaitez-vous continuer?</value>
+    <value>Vos données personnelles vont être anonymisées. Après cela, ces données ne seront plus reliées à votre identité et il ne vous ne sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```
Comment (valadas) [inline Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx]:
```suggestion
    <value>Vos données personnelles vont être anonymisées. Après cela, ces données ne seront plus reliées à votre identité et il ne vous sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```

File: Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx [hunk 2]
```diff
@@ -13,24 +13,24 @@
...
   <data name="cmdSubmit.Text" xml:space="preserve">
-    <value>Soumettre</value>
+    <value>Valider</value>
   </data>
   <data name="DataConsent.Text" xml:space="preserve">
     <value>J'accepte les &lt;a href="{0}"&gt;conditions d'utilisation&lt;/a&gt; et la &lt;a href="{1}"&gt;déclaration de confidentialité&lt;/a&gt; de ce site web.</value>
   </data>
   <data name="DelayedHardDelete.Confirm" xml:space="preserve">
-    <value>Vos données personnelles seront bientôt supprimées de ce site. Une fois qu'elles auront été supprimées, elles ne pouront plus être récupérées et vos données seront perdues à jamais. Vous ne pourrez également plus vous connecter avec ce compte. Souhaitez-vous continuer?</value>
+    <value>Vos données personnelles vont être supprimées de ce site très prochainement. Après cela elles seront définitivement perdues. De plus, il ne vous ne sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```
Comment (valadas) [inline Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx:28]:
```suggestion
    <value>Vos données personnelles vont être supprimées de ce site très prochainement. Après cela elles seront définitivement perdues. De plus, il ne vous sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```

File: Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx [hunk 3]
```diff
@@ -13,24 +13,24 @@
...
   </data>
   <data name="HardDelete.Confirm" xml:space="preserve">
-    <value>Vos données personnelles seront immédiatement supprimées de ce site. Une fois qu'elles auront été supprimées, elles ne pourront plus être récupérées et vos données seront perdues à jamais. Vous ne pourrez également plus vous connecter avec ce compte. Souhaitez-vous continuer?</value>
+    <value>Vos données personnelles vont être supprimées de ce site immédiatement. Après cela elles seront définitivement perdues. De plus, il ne vous ne sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```
Comment (valadas) [inline Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx]:
```suggestion
    <value>Vos données personnelles vont être supprimées de ce site immédiatement. Après cela elles seront définitivement perdues. De plus, il ne vous sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```

File: Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx [hunk 4]
```diff
@@ -13,24 +13,24 @@
...
   </data>
   <data name="ManualDelete.Confirm" xml:space="preserve">
-    <value>Nous nous efforcerons de supprimer vos données personnelles de ce site sous peu. Une fois qu'elles seront supprimées, elles ne pourront plus être récupérées et vos données seront perdues à jamais. Vous ne pourrez également plus vous connecter avec ce compte. Souhaitez-vous continuer?</value>
+    <value>Nous nous efforcerons de supprimer vos données personnelles de ce site très prochainement. Après cela elles seront définitivement perdues. De plus, il ne vous ne sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```
Comment (valadas) [inline Resources/DesktopModules/Admin/Security/App_LocalResources/DataConsent.ascx.fr-FR.resx:34]:
```suggestion
    <value>Nous nous efforcerons de supprimer vos données personnelles de ce site très prochainement. Après cela elles seront définitivement perdues. De plus, il ne vous sera plus possible de vous reconnecter avec ce compte. Souhaitez-vous continuer ?</value>
```

**Review by valadas — CHANGES_REQUESTED** [review]
Looks good except the noted corrections which you can approve in the browser if you agree.

**Comment by stetard** [comment]
Hi Daniel,
First, sorry for the mistakes.
I'm not sure about the process of validating your suggested changes. I hope it's correct now.
Just a question: yesterday, I spent few hours to try to complete the french language pack on a fresh DNN 9.13.3 install using the dedicated DNN interface. When I tried to copy the changes into the GitHub ressources repository, I got the surprise that the most was already done.
I'm wondering why the latest release didn't installed the latest french language pack. I think it could be better if the latest language pack is automaticaly provided. It seems that only the 9.2.0 is available in the proposed packages instead of the 9.9.1 which is better.

**Comment by valadas** [comment]
> I'm not sure about the process of validating your suggested changes. I hope it's correct now.

Exactly what you did, just accepting in the browser. For some more complex changes (like code that would also need testing and stuff) it could be additional commits pushed to the same branch.

> I'm wondering why the latest release didn't installed the latest french language pack

Because no release was created, I had no time to really test out the language pack since last release but also did not see anything crazy come in since then. If you feel this is 9.13.3 worth, I can create a release...

**Pull request merged**